substdi.c

```diff
@@ -43,12 +43,27 @@ ssize_t substdio_feed(substdio *s)
   return r;
 }
 
+#include "deprecated.h"
```

Comment:
This header is not needed here: it is already included in substio.h, which holds the prototype for this function.